src/core/mount.c

```diff
@@ -1916,6 +1916,18 @@ static void mount_handoff_timestamp(
         }
 }
 
+static void mount_notify_pidref(Unit *u, PidRef *parent_pidref, PidRef *child_pidref) {
```

Comment:
Is this really useful? I think we should try to restrict this to Service at least for now.

Reply:
yeah, i'd drop this too

Reply:
So how is this supposed to be done then? Move `PrivatePIDs=` to the service section? But we need it to be stored in `ExecContext`. Is there any prior art I can look at?